ui/idle.qml

```diff
@@ -72,6 +72,11 @@ Mycroft.CardDelegate {
         onIntentRecevied: {
             if (type == "phal.brightness.control.auto.night.mode.enabled") {
                 mainView.currentIndex = 0
+            } else if (type == "ovos.gui.main_view.current_index.set") {
```

Comment:
can we change this to use ovos.homescreen instead of ovos-gui ?

i'd like to keep ovos-gui namespace somewhat reserved for the ovos-gui repo itself

Reply:
of course! it makes more sense

updated